packages/view3d/test/suite/core/PostProcessing.ts

```diff
@@ -0,0 +1,68 @@
+import { createView3D } from "../../test-utils";
+
+describe("PostProcessing", () => {
+  describe("bloom effect", () => {
+    it("should resolution be greater than or equal to canvas size", async () => {
+      const view3d = await createView3D({bloom: true});
+
+      view3d.on('load', () => {
+        const resolutionSize = view3d.postProcessing.bloomComponent.resolution;
+        const canvasSize = view3d.renderer.canvasSize;
+
+        expect(resolutionSize.width >= canvasSize.width).to.be.true;
+        expect(resolutionSize.height >= canvasSize.height).to.be.true;
```

Comment:
Why does it have to be `greater than or equal`? The source code looks like setting it exactly the same as the renderer's rendering size.

Reply:
Becase of Mac resolution. 
https://github.com/mrdoob/three.js/issues/10238

Reply:
You can use `renderer.size` and strict equality(===) in that case, as that size includes devicePixelRatio.
of course, you can use `.to.equal()` instead too.

Reply:
Ok. Check it!